streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala

```diff
@@ -58,6 +58,12 @@ class StreamToTableJoinScalaIntegrationTestImplicitSerdes extends StreamToTableJ
     val userClicksStream: KStream[String, Long] = builder.stream(userClicksTopic)
 
     val userRegionsTable: KTable[String, String] = builder.table(userRegionsTopic)
+    userRegionsTable.filter { (_, _) =>
+      true
+    }
+    userRegionsTable.filterNot { (_, _) =>
+      false
+    }
```

Comment:
Once #5543 gets merged, we can implement a proper test a bit like https://github.com/apache/kafka/pull/5538/commits/b61cf73d3c228685edfd32003f3d0068b828b513#diff-def535526870ef4e9f8b5dcd83b812d5R32.